pyro/distributions/torch/normal.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import, division, print_function
+
+import torch
+
+from pyro.distributions.normal import Normal as _Normal
+from pyro.distributions.torch_wrapper import TorchDistribution
+from pyro.distributions.util import broadcast_shape, copy_docs_from
+
+
+@copy_docs_from(_Normal)
+class Normal(TorchDistribution):
+    reparameterized = True
+
+    def __init__(self, mu, sigma, *args, **kwargs):
+        torch_dist = torch.distributions.Normal(mu, sigma)
```

Comment:
so from what i can see, this returns the pt distribution which only provides sample and score methods.  but batch and event shapes eventually be pushed upstream as well, correct?

Reply:
We have batch and event shapes in `torch.distributions`, but they are internal attributes and not exposed as an API. Even in Pyro, we only need to use it, outside of the distribution instances, for testing. We still need to add analytical mean/variance to `torch.distributions` classes; otherwise, they should already be at feature parity.